website/source/layouts/guides.erb

```diff
@@ -50,6 +50,10 @@
         </ul>
       </li>
 
+      <li<%= sidebar_current("guides-tls") %>>
+        <a href="/guides/tls.html">Configuring TLS</a>
```

Comment:
What are you thoughts on "Securing Nomad" instead of "Configuring TLS"? The guide covers more than TLS, since it includes gossip, and I think we should optimize for people's googling.

If you agree, I'd also recommend renaming the file from /guides/tls.html.md to /guides/securing-nomad.html.md or /guides/securing-nomad-tls-encryption.html.md.